src/components/scenes/FioRequestConfirmationScene.js

```diff
@@ -123,16 +123,24 @@ export class FioRequestConfirmationConnected extends React.Component<Props, Stat
           if (getFeeRes.fee) {
             this.setState({ loading: false })
             this.resetSlider()
-            await Airship.show(bridge => (
+            const answer = await Airship.show(bridge => (
               <ButtonsModal
                 bridge={bridge}
                 title={s.strings.fio_no_bundled_err_msg}
+                message={s.strings.fio_no_bundled_add_err_msg}
                 buttons={{
-                  ok: { label: s.strings.string_ok }
+                  ok: { label: s.strings.title_fio_add_bundles }
                 }}
+                closeArrow
               />
             ))
-            // todo: redirect to 'add bundles' scene
+            if (answer === 'ok') {
+              navigation.navigate('fioAddressSettings', {
+                showAddBundles: true,
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name